nfsstat/metadata.csv

```diff
@@ -0,0 +1,10 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
This seems out of date:
the metrics are:
```
system.nfs.ops
system.nfs.read
system.nfs.read.exe
system.nfs.read.ops
system.nfs.read_per_s
system.nfs.read.retrans
system.nfs.read.retrans.pct
system.nfs.read.rtt
system.nfs.rpc_bklog
system.nfs.write
system.nfs.write.exe
system.nfs.write.ops
system.nfs.write_per_s
system.nfs.write.retrans
system.nfs.write.retrans.pct
system.nfs.write.rtt
```

Also I think the description are not always right (ie: my comments on the `check.py` for operation vs seconds).